discovery/src/main/java/com/newrelic/agent/discovery/TerminalColor.java

```diff
@@ -0,0 +1,34 @@
+package com.newrelic.agent.discovery;
+
+import java.util.logging.Level;
+
+public enum TerminalColor {
+    Red("\u001B[31m"),
```

Comment:
Is this enum actually used anywhere?

Reply:
Red (for error messages during attach), Yellow (for warn level messsages), and Cyan (for info level) are used.  

Green isn't used. I'll remove it